### PR TITLE
[ spi_host, proj ] SPI_HOST D2 milestone completion

### PR DESCRIPTION
**Pull request by martin-lueker**
- Update SPI_HOST to D2
- Update version to 1.0 for SPI_HOST design and DV

### CODE REVIEW

**Comment by sriyerg** [comment]
LGTM, thanks! Will defer to designers for approval.

**Review by rasmus-madsen — COMMENTED** [review]
LGTM - I think we need a designer as approver

File: hw/ip/spi_host/spi_host.core
Comment (tjaychen) [inline hw/ip/spi_host/spi_host.core:5]: i'm not super clear on the convention here, but i think the "1.0" here is technically supposed to correspond to an entry in `spi_host.prj.hjson`.  If that's the case, you might want to add a 1.0 entry there and indicate it is D2.  Kind of like [here](https://github.com/lowRISC/opentitan/blob/master/hw/ip/hmac/data/hmac.prj.hjson#L13).

Not a big deal in my opinion though.

Comment (tjaychen) [reply hw/ip/spi_host/spi_host.core:5]: looks good.

**Review by tjaychen — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]
Thanks @martin-lueker, LGTM (you need to sign off the first commit to pass CI).

**Pull request merged**
